src/Sarif.Converters.UnitTests/PylintConverterTests.cs

```diff
@@ -56,7 +56,8 @@ private static Result CreateResult()
             {
                 RuleId = "C0412(testSymbol)",
                 Message = new Message { Text = "testMessage" },
-                Level = ResultLevel.Warning,
+                Level = FailureLevel.Warning,
+                Kind = ResultKind.Fail,
```

Comment:
these pairs shouldn't literally be required after fixing up the property setters in Result.cs. i.e., simply by setting level to warning, kind will be updated properly. I'll fix these unnecessary inits later (it was my unease about requiring all these pairs to be kept accurate that led me to make the property changes)

Reply:
Yeah, I did comment on that. Not sure what's best, I leave it to you.

---
In reply to: [252549912](https://github.com/Microsoft/sarif-sdk/pull/1249#discussion_r252549912) [](ancestors = 252549912)